src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -76,8 +71,95 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
         approved: true
       }
     });
-  }
+    const wbs = await prisma.wBS_Element.findUnique({
+      where: {
+        wbsElementId: foundCR.wbsElementId
+      }
+    });
 
+    if (!wbs) {
+      return res.status(404).json({ message: `WBS element with id #${foundCR.wbsElementId} not found` });
+    }
+    const wp = await prisma.work_Package.findUnique({
+      where: {
+        workPackageId: wbs.workPackageNumber
```

Comment:
workpacakge number is not an id. we have WBS numbers that are like `1.2.3` so workpackagenumber would be the 3 as part of that. the include statement (including the project and wp on the query to get the wbs) from before was much better and more efficient